_topic_maps/_topic_map.yml

```diff
@@ -2999,6 +2999,8 @@ Topics:
       File: log6x-clf-6.2
     - Name: Configuring LokiStack storage
       File: log6x-loki-6.2
+    - Name: Configuring the logging collector
+      File: 6x-cluster-logging-collector-6.2
```

Comment:
Just checking that it is OK that this topic appears after "Configuring LokiStack storage" because in 6.0 and 6.1, it comes before that topic.

Reply:
That's an astute observation! Fixed it.